tools/test/h5diff/CMakeTests.cmake

```diff
@@ -533,11 +533,9 @@
               #-D "TEST_EXPECT=${resultcode}"
               -D "TEST_EXPECT=0" # ph5diff currently always exits with a zero status code due to
```

Comment:
Why does comment say `# ph5diff` for `tools/test/h5diff`?

Reply:
ph5diff runs the same tests as h5diff except in parallel mode.